packages/pulpcore/python-pulp-rpm/python-pulp-rpm.spec

```diff
@@ -12,28 +12,36 @@ Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{
 BuildArch:      noarch
 
 BuildRequires:  python3-devel
+Requires:       python3-PyGObject >= 3.22
+Conflicts:      python3-PyGObject >= 3.23
+BuildRequires:  python3-createrepo-c < 1.0
+BuildRequires:  python3-createrepo-c >= 0.15.10
+BuildRequires:  python3-django-readonly-field
+BuildRequires:  python3-jsonschema >= 3.0
+Requires:       python3-libcomps >= 0.1.11
+Conflicts:      python3-libcomps >= 0.2
+BuildRequires:  python3-productmd >= 1.25
+BuildRequires:  python3-pulpcore < 3.6
+BuildRequires:  python3-pulpcore >= 3.4
 BuildRequires:  python3-setuptools
+BuildRequires:  python3-solv
 
 %description
 %{summary}
 
 %package -n     python3-%{pypi_name}
 Summary:        %{summary}
 %{?python_provide:%python_provide python3-%{pypi_name}}
-%if 0%{?rhel} == 7
-Requires:       python36-gobject
-Requires:       libmodulemd2
-%else
-Requires:       python3-gobject
-Requires:       libmodulemd >= 2.0
-%endif
-Requires:       python3-createrepo_c < 1.0
-Requires:       python3-createrepo_c >= 0.15.10
+Requires:       python3-PyGObject >= 3.22
```

Comment:
Please restore the old requires (and build requires above), otherwise the package will end up uninstallable.